com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -35,6 +35,19 @@
     #define OUTPUT_SH(normalWS, OUT) OUT.xyz = SampleSHVertex(normalWS)
 #endif
 
+// WebGL1 (GLSL 1.0) cannot have variable for loop conditions. Use a constant
+// for loop with a break condition to get around this limitation.
+#define MAX_LIGHTS 8
+#if defined(SHADER_API_GLES) && !defined(SHADER_API_GLES3)
```

Comment:
Make it specific for WEBGL and GLES. GLES android GPUs support dynamic indexing.